### DISCUSSION TITLE
求解AdGuard Home二进制文件里首页文件是哪个，备案之后要求在首页填写备案号

### DISCUSSION THREAD

**Discussion by hengyv1998** [AdguardTeam/AdGuardHome#3970] [Q&A]
首页文件是哪一个 ，网站管理局要求填写编号

**Comment by ameshkov** [comment]
到首页添加网站备案编号你要自己改变 HTML 文件也编辑 AdGuard Home。

要改变这些 HTML 文件：https://github.com/AdguardTeam/AdGuardHome/tree/master/client/public
编辑说明：https://github.com/AdguardTeam/AdGuardHome#how-to-build

**Comment by Mosney** [comment]
居然真的有人头铁开DNS，并且还跑去备案
